README.md

```diff
@@ -69,6 +69,9 @@ myFunction:
       TABLE_NAME: my-table
     region: us-east-1
 
+    # if you want to use a specific role pass an arn
+    role: arn:XXX
```

Comment:
could we be a bit more specific here and call it `roleArn`, cause the user might pass a role component as well.

Reply:
@eahefnawy love it, I made the change